src/operations/Operations.jl

```diff
@@ -43,13 +110,13 @@ function calibrate(; model, dataset::DataFrame, context)
     fitp
 end
 
-"""
-NOT IMPLEMENTED
-"""
-function ensemble(; models::AbstractArray{AbstractPetriNet}, timespan=(0.0, 100.0)::Tuple{Float64, Float64})
+#-----------------------------------------------------------------------------# ensemble
+function ensemble(; models::AbstractArray{AbstractPetriNet}, timespan::Tuple{Float64, Float64} = (0.0, 100.0), context=nothing)
     throw("ENSEMBLE IS NOT YET IMPLEMENTED")
 end
 
+
+# joshday: what is this function?
```

Comment:
it looks like it's just `calibrate` but with a global optimization done instead. It should instead just be an option in `calibrate` set by the context.